components/molecules/MostActiveContributorsCard/most-active-contributors-card.tsx

```diff
@@ -217,16 +218,26 @@ function RowTooltip({
       <Tooltip.Trigger asChild>{children}</Tooltip.Trigger>
       <Tooltip.Portal>
         <Tooltip.Content sideOffset={-10} align="center" collisionPadding={10} side={"bottom"} avoidCollisions>
-          <div className={clsx("text-xs p-2 rounded shadow-lg bg-white font-light")}>
+          <div className="text-xs p-2 rounded shadow-lg bg-white font-light w-max">
             <div className="text-black font-bold mb-1">{contributor.login}</div>
             {Object.entries(dataLabelsList)
               .filter(([key]) => labels.includes(key))
-              .map(([key, value]) => (
-                <div key={key} className="flex items-center gap-2 font-bold">
-                  <div className="w-3 h-3 rounded-sm" style={{ backgroundColor: value.color }}></div>
-                  <div className="font-light">{value.title}</div>
-                </div>
-              ))}
+              .map(([key, value]) => {
+                const { percentage, stat } = userStats[key as StatKeys];
+
+                return (
+                  <div key={key} className="grid grid-cols-[1fr,1fr] gap-2 font-light">
```

Comment:
`grid-cols-2` it is not equivalent with `grid-cols-[1fr,1fr]` ??

Reply:
Good catch. Thanks @NsdHSO! I'm still new to Tailwind. 😅

Reply:
Done!